codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsRuntimeConfig.java

```diff
@@ -114,11 +116,12 @@ public void addConfigInterfaceFields(
                         + "trait was found on " + service.getId());
             }
         }
-        runtimeConfigs.putAll(getDefaultConfig(target));
+        runtimeConfigs.putAll(getDefaultConfig(target, isAwsService(settings, model)));
         return runtimeConfigs;
     }
 
-    private Map<String, Consumer<TypeScriptWriter>> getDefaultConfig(LanguageTarget target) {
+    private Map<String, Consumer<TypeScriptWriter>> getDefaultConfig(LanguageTarget target, boolean isAwsService) {
```

Comment:
I think I'd prefer this method just calculate `isAwsService` instead of taking in a new boolean flag. It has access to the same state as the caller, and it's private.